Bugzilla/Markdown.pm

```diff
@@ -65,13 +65,18 @@ sub render_html {
     return $html;
   }
 
-  $markdown =~ s{<(?!https?://)}{&lt;}gs;
+  # This is a bit faster since it doesn't engage the regex engine.
```

Comment:
this line of the comment is a bit out of place as the second line does indeed use a regex.  suggest deleting this line.